controllers/servers/host_definer/settings.py

```diff
@@ -47,3 +47,4 @@
 SUCCESSFUL_MESSAGE_TYPE = 'Successful'
 NORMAL_EVENT_TYPE = 'Normal'
 WARNING_EVENT_TYPE = 'Warning'
+FINALIZERS = 'finalizers'
```

Comment:
@matancarmeli7 could we add some context to `FINALIZERS`?

Reply:
`FINALIZERS_KEY` will be good?